lib/wcgi-runner/src/annotations.rs

```diff
@@ -0,0 +1,24 @@
+use wcgi_host::CgiDialect;
+
+// FIXME(@Michael-F-Bryan): Make this public in the webc crate
```

Comment:
Is there anything preventing us from making this public and releasing a new version of webc?

Better to not carry around this legacy burden, because it will spread to every runner, and also to other products we are working on...

But this also feeds into layering concerns about webc. 
See below for more.

Reply:
I've published this under the `webc::metadata::annotations` module.